canvas_exercise/shapes_game/index.js

```diff
@@ -1,36 +1,156 @@
 window.addEventListener("load", function() {
 
-  function clear(ctx, width, heigt) {
+  var shapeIndex = [0, 1, 2, 3];
```

Comment:
Since you're learning about `const` today, you could make some variables to keep track of what each one of these mean:

```
const ORANGE_RECT = 0;
const WHITE_RECT = 1;
const ORANGE_TRI = 2;
const WHITE_TRI = 3;
```